website/static/js/settings.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    WATERBUTLER_URL: 'http://localhost:7777/'
```

Comment:
Let's use the `WATERBUTLER_URL` variable already defined in `website.settings`. Yes, it will require using `contextVars` to use mako context in the JS. However, this is a small price to pay for greater consistency (easier refactoring in the future) and less repetition.